recipes/rodeo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Could we please add these arguments ( ` --single-version-externally-managed --record record.txt` )?